crates/sui/src/displays/dev_inspect.rs

```diff
@@ -24,19 +24,29 @@ impl<'a> Display for Pretty<'a, DevInspectResults> {
         write!(f, "{}", response.events)?;
 
         if let Some(results) = &response.results {
-            for result in results {
+            if results.is_empty() {
+                writeln!(f, "No execution results")?;
+                return Ok(());
+            } else {
```

Comment:
Nit: no `else` needed here -- you can just have the `writeln` after the `if`

Reply:
That makes sense, because we return early. Thank you!